test/operations/manager.go

```diff
@@ -495,12 +500,12 @@ func (m *Manager) SendL2Claim(ctx context.Context, deposit *pb.Deposit, smtProof
 	if err != nil {
 		return err
 	}
-	auth.GasPrice = big.NewInt(0)
+
+	// auth.GasLimit = 10 ^ 9        // TODO remove when the executor is fixed to estimate gas
+	auth.GasPrice = big.NewInt(0) // TODO remove when the executor is fixed to estimate gas
```

Comment:
I'm curious why you should remove it. In L2, claims must be performed using 0 gasPrice because a use could  have an empty account